servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@uronce-cc Yes, when the `BlockStoreLocation` is some determined `StorageDir`, `PARTIAL_LRU` is the same to `LRUEvictor`. `PARTIAL_LRU` is just a little logic changes to `LRUEvictor` overall.